std/string.d

```diff
@@ -482,12 +475,12 @@ ptrdiff_t indexOf(Range)(Range sIn, in dchar c,
     return -1;
 }
 
-ptrdiff_t indexOf(T, size_t n)(ref T[n] s, in dchar c,
-        in CaseSensitive cs = CaseSensitive.yes) @safe pure
-    if (isSomeChar!T)
+ptrdiff_t indexOf(Range)(auto ref Range s, in dchar c,
```

Comment:
The static array case is now handled by StringTypeOf as well. I'm puzzled how someone had seriously fixed on that particular overload.